app/assets/config/manifest.js

```diff
@@ -1,2 +1,3 @@
 //= link_tree ../images
 //= link_directory ../stylesheets .css
+//= link_directory ../javascripts .js
```

Comment:
Github wont let me comment on the image, so commenting just above! That image is 1.97 MB, 3000x2000 pixels. Is it necessary for it to be that large?